src/app/home/index.tsx

```
@@ -21,12 +24,15 @@
 import CommunityAni from "@/assets/community.json";
 import { useGetChartDataQuery } from "@/api/chartApi";
-import { useEffect, useState } from "react";
+import { useEffect, useRef, useState } from "react";
 import { useNavigate } from "react-router-dom";
 import { useModal } from "@/hooks/useModal.ts";
+import { SubscriptionModalContent } from "@/components/common/modal/SubscriptionModalContent";
 import { useAtom } from "jotai";
 import { loginState } from "@/store/user";
+import { useApiTotalView } from "@/hooks/api/visitor/useApiTotalView";
+import { VisitorCount } from "@/components/app/home/VisitorCount";
 import { SubscriptionModalContent } from "@/components/common/modal/SubscriptionModalContent";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'SubscriptionModalContent'. Consider to delete it or rename it.

'SubscriptionModalContent' is defined here:


(lint/suspicious/noRedeclare)